File: AMS.Web/src/app/models/manage-scheme/scheme-response.ts
import { OperationStatus } from "../shared/operation-status";
import { Scheme } from "./scheme";

export class SchemeResponse 
{
    //#region Constructor

    public SchemeResponse()
    {
        this.Scheme = new Array<Scheme>();
        this.OperationStatus = new OperationStatus();
    }

    //#endregion Constructor

    //#region Class Properties

    Scheme: Array<Scheme>;

    OperationStatus: OperationStatus;

    //#endregion
}

File: AMS.API/DTO/Scheme/SchemeResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AMS.API.DTO.Scheme
{
    public class SchemeResponse
    {
        #region Constructor

        public SchemeResponse()
        {
            Scheme = new List<Scheme>();
            OperationStatus = new OperationStatus();
        }
        #endregion Constructor

        #region Class Properties

        public List<Scheme> Scheme { get; set; }

        public OperationStatus OperationStatus { get; set; }

        #endregion
    }
}